sitemap.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset 
+    xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" 
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xmlns:xhtml="http://www.w3.org/1999/xhtml" 
+    xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9">
+    <url>
+        <loc>https://decisionhub.org/</loc>
+    </url>
+    <url>
+        <loc>https://decisionhub.org/support@decisionhub.org</loc>

```

Comment:
This seems weird. I noticed this was our email link awhile back, but I pushed an update recently to fix it. Are you on the latest version of master? (I changed this to `mailto:support@decisionhub.org`, but I think this may not even need to be in the sitemap anymore with that change)

Reply:
I am definitely on the latest version of master. When I run the website through https://app.seositecheckup.com/sitemap-generator it always gives me what I have. I tried running a free sitemap tool to compare and they didn't even include the <loc>https://decisionhub.org/support@decisionhub.org</loc>. I'm not sure what is correct.

Reply:
Well that link is definitely non-sensical since it doesn't go anywhere. So at the very least, let's remove it? But that is very strange..., I removed that link when I made that mailto change